monogen/cmd/main.go

```diff
@@ -17,11 +17,10 @@ var (
 		Long: "monogen scaffolds a Monomer project. " +
 			"The resulting project is compatible with the ignite tool (https://github.com/ignite/cli).",
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			return monogen.Generate(cmd.Context(), appDirPath, goModulePath, addressPrefix, skipGit, false)
+			return monogen.Generate(cmd.Context(), appDirPath, goModulePath, addressPrefix, "")
```

Comment:
Should `monomerPath` be passed in as the last param here instead of hardcoding it as `""`?